### PR TITLE
Add header styles

### PR DESCRIPTION
**Pull request by shekharkhedekar**
Adds styles for the mobile header - it looks _fine_ in desktop but needs some responsive love for some specific screen sizes.

Screenshot 2022-12-12 at 5 08 56 PM
Screenshot 2022-12-12 at 5 08 52 PM

### CODE REVIEW

File: wp-content/themes/vanguard-history/header.php [hunk 1]
```diff
@@ -31,7 +31,7 @@
 		if (! is_page($excluded_pages)){ ?>
         <header id="masthead" class="site-header ui">
             <div class="site-logo">
-                <img src="wp-content/themes/vanguard-history/img/Star.svg" />
+                <img src="<?php echo get_stylesheet_directory_uri(); ?>/img/Star.svg" alt="Vanguard star icon"/>
```
Comment (shekharkhedekar) [inline wp-content/themes/vanguard-history/header.php:34]: ah thank you

Comment (mrengy) [reply wp-content/themes/vanguard-history/header.php:34]: Sure thing. It was not loading the image locally for me before, so this seemed to fix it.

**Comment by mrengy** [comment]
Nice! Yeah, looking good. And not too bad on desktop either. I pulled this branch to dev at https://historyscv-dev.dreamhosters.com/ Only remaining issue I'm seeing is that there's a layer issue on the homepage - the navigation is appearing behind the embedded YouTube video https://jmp.sh/mv9ZheAM . Could you correct that @shekharkhedekar ?

**Comment by shekharkhedekar** [comment]
@mrengy done, thanks! I caught that but forgot to push the commit.

**Comment by mrengy** [comment]
Also I don't mind it personally, but the header background color is coming through for me as a dark brown in this branch. in https://xd.adobe.com/view/d26cac49-1b42-4c34-b329-59c06d191849-873f/specs/ it's showing #332F21 when I inspect it

**Comment by mrengy** [comment]
Wait, nevermind, looks as though that's what you implemented. for some reason it looks black in Adobe XD

**Comment by mrengy** [comment]
This is something that was lost in translation from wireframes to visual design file, but currently "Vanguard" and "History" wrap to two lines at 360 px width. The visual design file has it at 390 px, but I think we should make it fit in a single line on 360 at least.

Screen Shot 2022-12-13 at 4 23 12 PM

Glad you did the header first, so we can catch this before it imapcts the rest of the layout. So I'd suggest adjusting the rest of the layout to look as intended on 360px width as well.

**Review by mrengy — CHANGES_REQUESTED** [review]
Nice work! The layering issue is resolved with that update you pushed.

This is something that was lost in translation from wireframes to visual design file, but currently "Vanguard" and "History" wrap to two lines at 360 px width. The visual design file has it at 390 px, but I think we should make it fit in a single line on 360 at least.

Screen Shot 2022-12-13 at 4 23 12 PM

**Comment by shekharkhedekar** [comment]
Updated to handle 360px minimum width - and still look ok on larger screen sizes

**Review by mrengy — APPROVED** [review]
🎉

**Comment by mrengy** [comment]
All right. I deployed this to [stage](https://historyscv-stage.dreamhosters.com/year_story/2001-scv/). Also I renamed the dev and stage sites in WordPress settings so they both just have the site name of "Vanguard History" without the "[dev]" or "[stage]" before it - to make it match what production will do with the site name in that header.

Do you plan to continue to use this branch, or shall I delete it?

**Comment by shekharkhedekar** [comment]
I’ll delete.

**Comment by mrengy** [comment]
Thanks!

**Pull request merged**